### PR TITLE
fix: popup height, document plugin usage, error message show

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/class/ConnectError.js [hunk 1]
```diff
@@ -51,7 +51,12 @@ export default class ConnectError extends Error {
    */
   constructor(message, code = ERRORS.NOT_DEFINED) {
     super(message);
-    this.stack = new Error().stack;
+    this.name = 'ConnectError';
```
Comment (lamartire) [inline src/class/ConnectError.js]: Why we need `name` in Error?

File: src/class/ConnectError.js [hunk 2]
```diff
@@ -51,7 +51,12 @@ export default class ConnectError extends Error {
...
+    if (Error.captureStackTrace) {
```
Comment (lamartire) [inline src/class/ConnectError.js]: Does it really need?

Comment (lamartire) [reply src/class/ConnectError.js]: I think it is the default behaviour for any error

Comment (dudiq) [reply src/class/ConnectError.js]: in dev tools of chrome we can't see the message of error, that's why we use this solution.
without we can see only `uncautch Error: Error`, but with this we can see `uncautch Error: Error Connect library requires OAuth client id!` for example

File: src/class/ConnectError.js (hunk 1 above)
Comment (dudiq) [reply src/class/ConnectError.js]: Error should have name property for see in logs not `Error: some message`, but `ConnectError: some message`

File: src/class/ConnectError.js (hunk 2 above)
Comment (lamartire) [reply src/class/ConnectError.js]: Ok

**Comment by lamartire** [comment]
@kazurus approved

**Pull request merged**